packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -79,9 +87,28 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 				checked={ selected }
 			/>
 			<div className="domain-picker__suggestion-item-name">
-				<div>
+				<div className="domain-picker__suggestion-item-name-inner">
 					<span className="domain-picker__domain-name">{ domainName }</span>
 					<span className="domain-picker__domain-tld">{ domainTld }</span>
+					{ hstsRequired && (
+						<InfoTooltip position="middle right" noArrow={ false }>
+							{ createInterpolateElement(
+								__(
+									'All domains ending with <tld /> require an SSL certificate to host a website. When you host this domain at WordPress.com an SSL certificate is included. <learn_more_link>Learn more</learn_more_link>'
+								),
+								{
+									tld: <b>{ domainTld }</b>,
+									learn_more_link: (
+										<a
+											target="_blank"
+											rel="noreferrer"
+											href="https://wordpress.com/support/https-ssl"
+										/>
+									), // TODO Wrap this in `localizeUrl` from lib/i18n-utils
```

Comment:
Good TODO to add to https://github.com/Automattic/wp-calypso/issues/37665

Reply:
Done https://github.com/Automattic/wp-calypso/issues/45610 and added to https://github.com/Automattic/wp-calypso/issues/37665